src/core/agent/agent.types.ts

```diff
@@ -20,6 +20,15 @@ interface ConnectionHistoryItem {
 
 enum MiscRecordId {
   OP_PASS_HINT = "app-op-password-hint",
+  APP_ALREADY_INIT = "app-already-init",
+  APP_STATE_FLAGS = "app-state-flags",
+  APP_LANGUAGE = "app-language",
+  APP_IDENTIFIERS_FAVOURITES = "app-identifiers-favourites",
+  APP_CREDS_FAVOURITES = "app-creds-favourites",
+  APP_USER_NAME = "app-user-name",
+  APP_BIOMETRY = "app-biometry",
+  APP_KERIA_NOTIFICATION_MARKER = "app-keria-notification-marker",
+  APP_IDENTIFIER_VIEW_TYPE = "app-identifier-view-type",
```

Comment:
```
  APP_ALREADY_INIT = "app-already-init",
  APP_STATE_FLAGS = "app-state-flags",
  APP_LANGUAGE = "app-language",
  IDENTIFIER_FAVOURITES = "identifier-favourites",
  CREDENTIAL_FAVOURITES = "credential-favourites",
  USER_NAME = "user-name",
  APP_BIOMETRY = "app-biometry",
  KERIA_NOTIFICATION_MARKER = "keria-notification-marker",
  APP_IDENTIFIER_VIEW_TYPE = "app-identifier-view-type",
```

Can we update to this? I'm not sure why `APP_` and `app-` got in front of all of these. But it makes sense to have it in front of language to signify it's a setting/flag.

Reply:
@iFergal Will I change prefix to setting. Ex: setting-user-name

Reply:
I think just `user-name` is OK for the username

Reply:
@iFergal Will i update only username field, or all field?

Reply:
All fields - just copy what I wrote :P 

```
  APP_ALREADY_INIT = "app-already-init",
  APP_STATE_FLAGS = "app-state-flags",
  APP_LANGUAGE = "app-language",
  IDENTIFIER_FAVOURITES = "identifier-favourites",
  CREDENTIAL_FAVOURITES = "credential-favourites",
  USER_NAME = "user-name",
  APP_BIOMETRY = "app-biometry",
  KERIA_NOTIFICATION_MARKER = "keria-notification-marker",
  APP_IDENTIFIER_VIEW_TYPE = "app-identifier-view-type",
```